src/rust/ide/src/controller/searcher.rs

```diff
@@ -354,10 +374,10 @@ impl Data {
                 id                : CompletedFragmentId::Function,
                 picked_suggestion : entry
             };
-            // When editing node, we don't have any special handling for "this" node.
-            // This might be revisited in the future.
-            let this_var = None;
-            fragment.is_still_unmodified(&input,this_var,&current_module).and_option(Some(fragment))
+            // This is meant to work with single function calls (without this argument).
```

Comment:
the part `without this argument` is easily misleading, I would put the word "this" in backticks or quotation marks, as it was before.